rest_api/sawtooth_rest_api/route_handlers.py

```diff
@@ -262,7 +262,7 @@ def __init__(
             paging: Paging info and nav, like total resources and a next link
         """
         paging_controls = self._get_paging_controls(request)
-
+        self._get_head_id(request)
```

Comment:
The return value should be passed to `_head_to_root`.  This now hides the fact that you're calling that to validate the input.  A future developer may come along and delete that line, thinking its extraneous.